scripts/update-crd-reference/config.yaml

```
@@ -268,7 +268,7 @@ source_repositories:
         hidden: true
       certconfigs.core.giantswarm.io:
         owner:
-          - https://github.com/orgs/giantswarm/teams/team-cabbage
+          - https://github.com/orgs/giantswarm/teams/team-phoenix
```

```suggestion
          - https://github.com/orgs/giantswarm/teams/team-hydra
```
according to the `Ownership of components` spreadsheet